Makefile

```diff
@@ -1,43 +1,36 @@
-SPEL_DESC_URL ?= https://github.com/plus3it/spel
-SPEL_AMIGEN7SOURCE ?= https://github.com/plus3it/AMIgen7.git
-SPEL_AMIGEN7BRANCH ?= master
-SPEL_AMIGEN7REPOSRC ?= https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm,https://spel-packages.cloudarmor.io/spel-packages/repo/spel-release-latest-7.noarch.rpm
-SPEL_AMIGEN7REPOS ?= spel
-SPEL_AMIGEN8SOURCE ?= https://github.com/plus3it/AMIgen8.git
-SPEL_AMIGEN8BRANCH ?= master
-SPEL_AMIGEN8REPOSRC ?= https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm,https://spel-packages.cloudarmor.io/spel-packages/repo/spel-release-latest-8.noarch.rpm
-SPEL_AMIGEN8REPOS ?= spel
-SPEL_AWSCLIV1SOURCE ?= https://s3.amazonaws.com/aws-cli/awscli-bundle.zip
-SPEL_AWSCLIV2SOURCE ?= https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip
-SPEL_AMIGENBUILDDEV ?= /dev/nvme0n1
-SPEL_EPEL7RELEASE ?= https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-SPEL_EPEL8RELEASE ?= https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
-SPEL_EPELREPO ?= epel
-SPEL_EXTRARPMS ?= python36,spel-release,ec2-hibinit-agent,ec2-instance-connect,ec2-net-utils,https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
-SOURCE_AMI_CENTOS7_HVM ?= ami-00e87074e52e6c9f9
-SOURCE_AMI_RHEL7_HVM ?= ami-073955d8665a7a9e7
-SOURCE_AMI_CENTOS8STREAM_HVM ?= ami-0ee70e88eed976a1b
-SOURCE_AMI_RHEL8_HVM ?= ami-06644055bed38ebd9
-SSH_INTERFACE ?= public_dns
-PIP_URL ?= https://bootstrap.pypa.io/get-pip.py
-PYPI_URL ?= https://pypi.org/simple
-SECURITY_GROUP_CIDR := $(shell curl -sSL 'https://api.ipify.org')/32
+SPEL_BUILDERS ?= amazon-ebs.minimal-rhel-7-hvm,amazon-ebs.minimal-centos-7-hvm,amazon-ebs.minimal-rhel-8-hvm,amazon-ebs.minimal-centos-8stream-hvm
+export PATH := $(HOME)/.local/bin:$(PATH)
+
+# The `pre_build`, `build`, and `post_build` targets all use packer in a way that
+# supports both Commercial and GovCloud partitions. For GovCloud, the `install`
+# target is used to setup an aws profile with credentials retrieved from SSM. For
+# the Commercial partition, the profile is created but the credentials are sourced
+# from the execution environment (meaning your workstation or CodeBuild).
+
+# Due to the use of an aws profile, when running interactively, it is required
+# to export AWS_PROFILE with a valid profile. For CodeBuild CI, it is set to $SPEL_IDENTIFIER,
+# and `make install` will create it.
+
+# For GovCloud, set the ami filters to find the correct AMIs. The default values
+# otherwise work fine for all RedHat build (GovCloud or otherwise), and for all
+# builds in the Commercial partition.
+ifeq ($(PKR_VAR_aws_region),us-gov-west-1)
+PKR_VAR_aws_source_ami_filter_centos7_hvm = {name = "*-Recovery (No-LVM)-ACB-CentOS7-HVM-SRIOV_ENA", owners = ["039368651566"]}
+PKR_VAR_aws_source_ami_filter_centos8stream_hvm = {name = "spel-bootstrap-centos-8stream-hvm-*.x86_64-gp2", owners = ["039368651566"]}
```

Comment:
Will having the account number hard-coded this way break cross-partition portability?

Reply:
Not really, as it is conditioned on the env `PKR_VAR_aws_region`. It works fine specifically for us-east-1 (which use the default values) and us-gov-west-1 (this specific override). So technically, this is actually here to *support* cross-partition, but only *those* two partitions. 

We also haven't been using the Makefile in iso regions, preferring just to run the packer cli directly and pass the vars.

Reply:
At some point, do we want to look at GitLab-CI (and/or GitHub Actions) recipes for partitions where CodeBuild isn't available?

Reply:
I'm expecting that would be managed in a separate repo, local to that partition. The repo could probably be just a ci config that clones the spel repo and runs packer.